core.Dockerfile

```
@@ -58,7 +58,7 @@ ADD test/            test/
 ADD .git/            .git/
 ADD .gitattributes Makefile Makefile.venv go.mod go.sum .
 
-RUN make innerbuild GOOS=$TARGETOS GOARCH=$TARGETARCH GIT_DIRTY=$GIT_DIRTY IGNORE_GO_VERSION=yesplease
+RUN make innerbuild TARGETOS=$TARGETOS TARGETARCH=$TARGETARCH GIT_DIRTY=$GIT_DIRTY IGNORE_GO_VERSION=yesplease
```

This PR adds complexity that, I think, is not required.  Ask yourself why the Makefile has an `OS` variable at all. It seems to me so to be there so that the `make` user can override the target platform. So we do not need to introduce _another_ pair of variables to support setting the target platform.

```suggestion
RUN make innerbuild OS=$TARGETOS ARCH=$TARGETARCH GIT_DIRTY=$GIT_DIRTY IGNORE_GO_VERSION=yesplease
```